dvl:
  ftdi: D309SFWS
  negate_x_vel: false
  negate_y_vel: false
  negate_z_vel: false
ping1D:
  ftdi: DP05ZD1V
pv:
  ftdi: B0015XL0
servo:
  ftdi: A10N1UC1